cyberdrop_dl/crawlers/transflix.py

```diff
@@ -51,7 +51,7 @@ async def video(self, scrape_item: ScrapeItem) -> None:
         video = css.select_one(soup, _SELECTORS.VIDEO)
         link = self.parse_url(css.get_attr(video, "src"))
         filename, ext = self.get_filename_and_ext(link.name)
-        scrape_item.possible_datetime = int(Path(filename).stem)
+        scrape_item.possible_datetime = _timestamp_from_filename(filename)
```

Comment:
Maybe also a good idea to use the raw filename

The proceed filename may have been truncated by config values
```suggestion
        scrape_item.possible_datetime = _timestamp_from_filename(link.name)
```